src/main/java/pl/zimi/zadanka/Zad6.java

```diff
@@ -6,35 +6,28 @@ public int removeDuplicates(int[] nums) {
         if (nums.length == 1) return 1;
 
         int numsLen = nums.length - 1;
-        int numsMin = nums[0];
         int numsMax = nums[numsLen];
         int uniqueNumbersCounter = 0;
-        int tempPreviousNumber = 1000000;
+        int tempPreviousNumber = -1000000;
```

Comment:
jeśli chcesz użyć minimalnej wartości użyj Integer.MIN_VALUE zamiast takiej "magic constant"
magic constant czyli jakaś arbitralnie ustalona liczba czy string który się pojawia w kodzie są anty wzorcem
tworzymy stałą z wartością i ew. tą stała przypisujemy

Reply:
Wiesz, co.... ubzdurałem sobie, że gdy wartość bezwzględna liczby będzie zbyt duża to spowoduje obniżenie szybkości wykonywania. Nie wiedziałem, że jest istnieje Integer.max/min będę to stosowal